workshops/kuizzy/README.md

```diff
@@ -0,0 +1,659 @@
+---
+name: Kahoot Clone - Kuizzy
+description: Make a Kahoot clone with SweetAlert, WebSockets, and Node.js
+author: '@KhushrajRathod'
+img: https://cloud-b0r4v3fgu.vercel.app/0screenshot_2020-12-31_at_3.22.47_pm.png
+---
+
+Love [Kahoot](https://kahoot.com/)? I do! Today were making a Kahoot clone -- Kuizzy -- using [SweetAlert](https://sweetalert.js.org/), a JS alert library, [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) and [Node.js](https://nodejs.org/).
```

Comment:
I think we should have a bit more detail about what WebSockets & SweetAlert is at the start, if that makes sense?